data_structures/linked_list/cycle_in_linked_list.c

```diff
@@ -0,0 +1,97 @@
+/**
+ * @file
+ * @author [Ankita Kulkarni](https://github.com/ankitakul)
+ * @brief This Algorithm detects a loop in a singly linked list 
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂